tests/spread/integration/netcat-openbsd/task.yaml

```diff
@@ -0,0 +1,9 @@
+summary: Integration tests for netcat-openbsd
+
+execute: |
+  rootfs="$(install-slices netcat-openbsd_bins)"
+
+  chroot "${rootfs}/" nc.openbsd -h
+  chroot "${rootfs}/" nc.openbsd -l 1234 >testfile &
+  echo "Hello World" | chroot "${rootfs}/" nc.openbsd -q 1 -w 1 localhost 1234
+  test "$(cat "${rootfs}/testfile")" = "Hello World"
```

Comment:
Oops, I think this should actually be the following, I forgot the shell redirection occurs outside the chroot:
```shell
  chroot "${rootfs}/" nc.openbsd -h
  chroot "${rootfs}/" nc.openbsd -l 1234 >testfile &
  echo "Hello World" | chroot "${rootfs}/" nc.openbsd -q 1 -w 1 localhost 1234
  test "$(cat testfile)" = "Hello World"
```